docs/sources/tempo/traceql/_index.md

```diff
@@ -73,8 +72,32 @@ In TraceQL, curly brackets `{}` always select a set of spans from the current tr
 
 ### Intrinsic fields
 
-Intrinsic fields are fundamental to spans. These fields can be referenced when selecting spans.
-Custom attributes are prefixed with `.`, `span.` or `resource.`, whereas intrinsics are typed directly.
+Intrinsic fields are fundamental to scopes.
+Intrinsics are inherently present, as opposed to other key-value pairs (attributes) that are added by a developer.
+
+Intrinsics are always indicated using a `<scope>:`.
```

Comment:
should this all go under a "Intrinsics vs Attributes" heading?

Reply:
I know where I can move it. I'll update shortly.